### PR TITLE
Start proxying only specific network operations

### PR DESCRIPTION
**Pull request by kasperl**

### CODE REVIEW

File: system/extensions/host/network.toit [hunk 1]
```diff
@@ -24,19 +24,9 @@ class NetworkServiceDefinition extends NetworkServiceDefinitionBase:
...
-      socket.connect
-        net.SocketAddress
-          net.IpAddress.parse "8.8.8.8"
-          80
-      return socket.local_address.ip.to_byte_array
-    finally:
-      socket.close
+  connect client/int -> List:
+    resource := NetworkResource this client
+    return [resource.serialize_for_rpc, 0]
```
Comment (floitsch) [inline system/extensions/host/network.toit:29]: The 0 doesn't give a lot of information. Maybe this way?
```suggestion
    proxy_bits := 0
    return [resource.serialize_for_rpc, proxy_bits]
```

File: tests/services_network_test.toit [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    return (open_ FakeNetworkService.UUID FakeNetworkService.MAJOR FakeNetworkService.MINOR) and this
+
+class FakeNetworkServiceDefinition extends ServiceDefinition:
+  proxy_mask_/int := 0
+
+  address_/ByteArray? := null
+  resolve_/List? := null
+
+  constructor:
+    super "system/network/test" --major=1 --minor=2
```
Comment (floitsch) [inline tests/services_network_test.toit:71]: If the values for major and minor are not important add a comment.

**Review by floitsch — APPROVED** [review]
LGTM.

File: tests/services_network_test.toit (hunk 2 above)
Comment (kasperl) [reply tests/services_network_test.toit:71]: Done.

File: system/extensions/host/network.toit (hunk 1 above)
Comment (kasperl) [reply system/extensions/host/network.toit:29]: Done.

**Pull request merged**